apps/service_providers/llm_service/main.py

```diff
@@ -168,6 +172,15 @@ def transcribe_audio(self, audio: BytesIO) -> str:
         )
         return transcript.text
 
+    def attach_built_in_tools(self, built_in_tools: list[str], tools: list[BaseTool]):
```

Comment:
What if `built_in_tools` is a list of type `BuiltInTools`. If we have a mapping in `BuiltInTools` that returns the tool type, it might be a tiny bit easier to manage. Here's what I mean

```python
class BuiltInTools(models.TextChoices):
    WEB_SEARCH = "web-search", gettext("Web Search")
    CODE_EXECUTION = "code-execution", gettext("Code Execution")


    @property
    def tool_type(self) -> str:
        if self == BuiltInTools.WEB_SEARCH:
            return "web-search-preview"
        ...
```

Reply:
Oh I'm only now seeing [this comment](https://github.com/dimagi/open-chat-studio/pull/1579/files#r2086236993) that is similar